cmd/registry/types/types.go

```diff
@@ -71,6 +71,19 @@ func IsZipArchive(mimeType string) bool {
 	return strings.Contains(mimeType, "+zip")
 }
 
+// IsPrintableType returns true if the corresponding contents can be treated as a string.
+func IsPrintableType(mimeType string) bool {
+	if mimeType == "text/plain" {
+		return true
+	} else if strings.HasPrefix(mimeType, "application/yaml") {
+		return true
+	} else if strings.HasPrefix(mimeType, "application/json") {
+		return true
+	} else {
+		return false
+	}
```

Comment:
`return mimeType == "text/plain" || strings.HasPrefix(mimeType, "application/yaml") || strings.HasPrefix(mimeType, "application/json")`

Reply:
I find this easier to read and more extensible than one long line, but ok.

Reply:
Updated, but since the single-line expression was over 130 columns wide, I included line breaks.